rubies/truffleruby-gftc-dev

```diff
@@ -0,0 +1,19 @@
+platform="$(uname -s)-$(uname -m)"
+case $platform in
+Linux-x86_64)
+  install_package "truffleruby-head" "https://github.com/graalvm/oracle-graalvm-dev-builds/releases/latest/download/truffleruby-dev-linux-amd64.tar.gz" truffleruby
```

Comment:
Should the `"truffleruby-head"` part be `"truffleruby-gftc-dev"` in all of these definitions?